include_core/omrutil.h

```diff
@@ -300,7 +300,25 @@ omr_error_t detectVMDirectory(wchar_t *vmDirectory, size_t vmDirectoryLength, wc
 
 uintptr_t getStorageKey(void);
 
-#endif /*if defined(J9ZOS390)*/
+/* ---------------- zosversion.c ---------------- */
+
+#define ZOS_V1R10_RELEASE 20.00
+#define ZOS_V1R10_VERSION 3
+
+#define ZOS_V2R4_RELEASE 27.00
+#define ZOS_V2R4_VERSION 4
+
+/**
+ * Function to determine if the zos version is at least a given
+ * release and version.  The implementation is based on uname(),
```

Comment:
Are the two spaces intentional?
```suggestion
 * release and version. The implementation is based on uname(),
```